config/mbed/CMakeLists.txt

```diff
@@ -139,9 +139,9 @@ endif()
 #chip_gn_arg_cflags("target_cflags"                          ${CHIP_CFLAGS})
 #chip_gn_arg_cflags("target_cflags_c"                        ${CHIP_CFLAGS_C})
 #chip_gn_arg_cflags("target_cflags_cc"                       ${CHIP_CFLAGS_CC})
-chip_gn_arg_string("mbed_ar"                              ${CMAKE_AR})
-chip_gn_arg_string("mbed_cc"                              ${CMAKE_C_COMPILER})
-chip_gn_arg_string("mbed_cxx"                             ${CMAKE_CXX_COMPILER})
+#chip_gn_arg_string("mbed_ar"                              ${CMAKE_AR})
+#chip_gn_arg_string("mbed_cc"                              ${CMAKE_C_COMPILER})
+#chip_gn_arg_string("mbed_cxx"                             ${CMAKE_CXX_COMPILER})
```

Comment:
There is no need to comment on this if you are not creating a file (line 162). Anyway, I also changed this file significantly adding a compilation flag etc. Therefore the easiest way will be to merge these changes then I'll add mine and overwrite these a bit.